specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyEvents.json

```diff
@@ -603,6 +615,10 @@
           "format": "int32",
           "minimum": 0
         },
+        "@odata.nextLink": {
```

Comment:
>nextLink [](start = 16, length = 8)

You're missing a lot of x-ms-pageable declarations in this file. (The easiest way to find them is to manually diff 2018-07-01-preview version of this file with this one.)

Reply:
Thank you had missed that, have added it now